README.md

```diff
@@ -11,11 +11,16 @@ The ODF CLI tool provides configuration and troubleshooting commands for OpenShi
   - `recovery-profile`: Get the recovery profile value.
   - `health`: Check health of the cluster and common configuration issues.
 - `odf purge-osd <ID>`: Permanently remove an OSD from the cluster.
-- `odf help` : Display help text
-- `odf subvolume ls`: Display all the subvolumes
-- `odf subvolume delete <subvolume> <filesystem> <subvolumegroup>`: Deletes the stale subvolumes
+- `odf maintenance`: [Perform maintenance operations](docs/maintenance.md) on mons or OSDs. The mon or OSD deployment will be scaled down and replaced temporarily by a maintenance deployment.
+  - `start <deployment-name>`
+    - `[--alternate-image <alternate-image>]` : Start a maintenance deployment with an optional alternative ceph container image
+  - `stop <deployment-name>`: Stop the maintenance deployment and restore the mon or OSD deployment
+- `odf subvolume`:
+  - `ls`: Display all the subvolumes
+  - `delete <subvolume> <filesystem> <subvolumegroup>`: Deletes the stale subvolumes
 - `odf mons`
   - `restore-quorum <monID>`: Restore the mon quorum based on a single healthy mon since quorum was lost with the other mons
+- `odf help` : Display help text
```

Comment:
displays all the odf cli commands with a help text

Reply:
it doesn't only display the command but flags, args, and root flags. So, it is standard to mention help text.